server/routes/contacts/add/relationship-comments/enterRelationshipCommentsController.ts

```diff
@@ -12,11 +12,13 @@ export default class EnterRelationshipCommentsController implements PageHandler
     const { journeyId } = req.params
     const journey = req.session.addContactJourneys[journeyId]
     const view = {
+      contact: journey.names,
```

Comment:
on the other common pages we do this the other way around and inject `names` into the standalone journey. `journey: { ...journey, names },` see UpdateEstimatedDobController. I would prefer to keep consistent with this.

Reply:
updated to use journey.names to be consistent